src/plugins/intel_cpu/tests/functional/custom/single_layer_tests/classes/conversion.cpp

```diff
@@ -63,7 +63,17 @@ void ConvertCPULayerTest::SetUp() {
     auto primitive = selectedType;
     if (primitive.empty())
         primitive = getPrimitiveType();
-    if (!isInOutPrecisionSupported(inPrc, outPrc))
+#if defined(OPENVINO_ARCH_ARM64)
```

Comment:
There is method `isInOutPrecisionSupported` which exclude some test cases for acl on arm. Should we remove them if now `Convert` will be executed via Snippets?

Reply:
Thanks Alexandra for the comment! You are right now Convert will be executed via Snippets. Yet tokenization of Snippets has other contraints besides precision. For example, thought i8 is generally supported by Snippets, but some i8 cases will not be tokenized because of for exmaple rank discussed in next comment. For such case, we still need to return false of isInOutPrecisionSupported for acl, when it does not support.

And as I added `primitive != "jit"` to the condition, isInOutPrecisionSupported will not be executed if primitive already equal to "jit". Please feel free to have further discussions.